### DISCUSSION TITLE
Questions on wasm worker

### DISCUSSION THREAD

**Discussion by msqr1** [emscripten-core/emscripten#21610] [Q&A]
What is the difference between the wasm worker and the worker api in emscripten.h
How to know how much stack size of a wasm worker? 
What does a wasm worker do when it runs out of that stack?
Why do I have to know how much a wasm worker need at compile time? Can it just malloc?

**Comment by msqr1** [comment]
@sbc100 @juj

**Comment by juj** [comment]
Good questions: I posted https://github.com/emscripten-core/emscripten/pull/21686 to improve the documentation on these aspects. LMK if there's wordings that could be improved there, or things to expand on further.

**Comment by msqr1** [comment]
Oh kay
